exercises/1-the-manual-menace/README.md

```diff
@@ -436,7 +436,21 @@ where
     * `<GIT_URL>` is the full clone path of the repo where this project is stored (including the https && .git)
     * `<YOUR_NAME>` is the prefix for your `-ci-cd` project.
 
-8. Create a new object `ci-cd-builds` in the Ansible `inventory/host_vars/ci-cd-tooling.yml` to drive the s2i build configuration.
+8. Create `params/jenkins-s2i-slaves` and add the following content; replacing variables as appropriate.
+
+<kbd>📝 *enablement-ci-cd/params/jenkins-s2i-slaves*</kbd>
+```
+IMAGE_STREAM_NAME=jenkins-slave-npm
+IMAGE_STREAM_NAMESPACE=<YOUR_NAME>-ci-cd
+IMAGE_STREAM_ROLE=jenkins-slave
```

Comment:
If we use Params from vars, we can not have people edit YOUR_NAME and also get reuse of the string `jenkins-slave-npm` by setting it to a variable

Reply:
agreed